src/components/tables/Table.tsx

```diff
@@ -51,6 +52,11 @@ export interface ITablePredicate {
   attributeNameFormatter?: IAttributeNameOrValueFormatter;
 }
 
+export interface ITableDatePicker extends IDatePickerDropdownProps {
+  // If the table is not manual, tells which attribute should be used to filter dates
+  attributeValue?: string;
```

Comment:
I'm wondering if this attribute should be in `IDatePickerDropdownProps` ?

Reply:
Good point!

Reply:
should be `attributeName` and not `attributeValue` right?